src/main/java/seedu/address/model/item/Item.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.model.item;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+/**
+ * Represents an Item in the Expense Splitter of EYLAH.
+ * Guarantees: details are present and not null.
+ */
+public class Item {
+
+    // Identity fields
+    private final ItemName name;
+
+    // Data fields
+    private final ItemPrice price;
+
+    /**
+     * Every field must be present,
+     * ItemName and ItemPrice fields must not be null.
+     */
+    public Item(ItemName name, ItemPrice price) {
+        requireAllNonNull(name, price);
+        this.name = name;
+        this.price = price;
+    }
+
+    public ItemName getName() {
```

Comment:
Maybe you should change method name to `getItemName` to provide a better description for the method.

Reply:
Alright understood. thank you